net.mediaarea.BWFMetaEdit.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016-2017 MediaArea.net <info@mediaarea.net> -->
+<component type="desktop">
+  <id type="desktop">net.mediaarea.BWFMetaEdit.desktop</id>
+  <metadata_license>CC-BY-3.0</metadata_license>
+  <project_license>Public Domain</project_license>
+  <name>BWF MetaEdit</name>
+  <summary>Embed, validate, and export BWF files metadata</summary>
+  <description>
+    <p>BWF MetaEdit was developed by the Federal Agencies Digitization Guidelines Initiative (FADGI) supported by AudioVisual Preservation Solutions.</p>
+    <p>This tool permits embedding, editing, and exporting of metadata in Broadcast WAVE Format (BWF) files. This tool can also enforce metadata guidelines developed by the Federal Agencies Audio-Visual Working Group, as well as recommendations and specifications from the European Broadcasting Union (EBU), Microsoft, and IBM.</p>
+  </description>
```

Comment:
It'd be great if you could include one or more screenshots here, and ideally release tags as well. We're also asking apps to include [OARS metadata](https://odrs.gnome.org/oars) if they can (should be very easy for you! lots of 'no this app doesn't do this).